services/venice-controller/src/main/java/com/linkedin/venice/controller/InstanceStatusDecider.java

```diff
@@ -32,29 +32,20 @@ static List<Replica> getReplicasForInstance(HelixVeniceClusterResources resource
   }
 
   static NodeRemovableResult isRemovable(HelixVeniceClusterResources resources, String clusterName, String instanceId) {
-    return isRemovable(resources, clusterName, instanceId, Collections.emptyList(), false);
+    return isRemovable(resources, clusterName, instanceId, Collections.emptyList());
   }
 
   private static PartitionAssignment removeLockedResources(
       PartitionAssignment partitionAssignment,
       List<String> lockedNodes,
-      String resourceName,
-      boolean isInstanceView) {
+      String resourceName) {
     PartitionAssignment assignment = partitionAssignment;
     for (String lockedNodeId: lockedNodes) {
-      assignment = getPartitionAssignmentAfterRemoving(lockedNodeId, assignment, resourceName, isInstanceView);
+      assignment = getPartitionAssignmentAfterRemoving(lockedNodeId, assignment, resourceName, false);
```

Comment:
Can we add some comment here why we are passing instanceView false for locked nodes.

Reply:
Sure. Will add one